PhpStorm Protocol (Win)/run_editor.js

```diff
@@ -22,6 +27,10 @@ if (match) {
 		file_system = new ActiveXObject('Scripting.FileSystemObject'),
 		file = decodeURIComponent(match[1]).replace(/\+/g, ' '),
 		search_path = file.replace(/\//g, '\\');
+		
+	if (settings.project_basepath != '' && settings.project_pathalias != '') {
+		file = file.replace(settings.project_basepath, settings.project_pathalias);

```

Comment:
> If I replace it with an empty string in Windows, this results in the following exception:

That maybe due other lines in that `run_editor.js` script. Always hated Windows.

You can change other script parts as needed to make relative path opening work.